functorch/docs/source/install.rst

```diff
@@ -1,29 +1,28 @@
 Install functorch
 =================
 
-pip
----
-
-To install functorch via pip, please first install
-`PyTorch 1.11 <https://pytorch.org/get-started/locally/>`_
-and then run the following command:
-
-::
-
-  pip install functorch
-
-We currently support manylinux, x86 MacOS, and Windows, please reach out on
-`GitHub <https://github.com/pytorch/functorch>`_ for other platforms.
+As of PyTorch 1.13, functorch is now included in the PyTorch binary. That is,
+installing PyTorch will let you ``import functorch``. Please see the
+`PyTorch homepage <https://pytorch.org>`_ for installation instructions.
 
 Colab
 -----
 
 Please see `this colab for instructions. <https://colab.research.google.com/drive/1GNfb01W_xf8JRu78ZKoNnLqiwcrJrbYG#scrollTo=HJ1srOGeNCGA>`_
```

Comment:
Do we need to update the colab? Or just say to use PyTorch after 1.13?

Reply:
I will update the colab after the release, since it's linked from our (0.2.1) docs. Note that `pip install functorch` will still work in pytorch 1.13, so the colab will still technically be correct